orc8r/cloud/go/service/middleware/unary/identity_decorator_bypass.go

```diff
@@ -17,6 +17,8 @@ package unary
 // bypass Identity verification checks.
 // For now, the only service in this category is Bootstrapper.
 var identityDecoratorBypassList = map[string]struct{}{
-	"/magma.orc8r.Bootstrapper/GetChallenge": {},
-	"/magma.orc8r.Bootstrapper/RequestSign":  {},
+	"/magma.orc8r.Bootstrapper/GetChallenge":              {},
+	"/magma.orc8r.Bootstrapper/RequestSign":               {},
+	"/magma.orc8r.Registration/Register":                  {},
+	"/magma.orc8r.CloudRegistration/GetGatewayDeviceInfo": {},
```

Comment:
Shouldn't need to add this here -- "CloudRegistration" is called by "Registration", rather than directly from gw

<img width="1318" alt="Screen Shot 2022-02-23 at 2 32 47 PM" src="https://user-images.githubusercontent.com/8029544/155403557-aa078ce5-3f94-4c6a-9677-db46f0ce99e0.png">